tpl/page_optm/settings_html.tpl.php

```diff
@@ -87,6 +87,26 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_OPTM_HTML_SKIP_COMMENTS; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_textarea($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('Keep html comments.', 'litespeed-cache'); ?>
```

Comment:
Not sure what this setting does. Is it keeping certain comments from being minified when you minify the HTML? I want to change the description to make it more clear, but first I want to make sure I understand what the setting is for.

Reply:
Changed the text, please confirm :) thanks